src/frontend/src/app/[locale]/(auth)/dashboard/documents/page.tsx

```diff
@@ -0,0 +1,501 @@
+'use client';
+import { useState, useEffect } from 'react';
```

Comment:
When I change organisation, the files for the last organisation is displayed
Ideally we should fetch the files for the current org and then display the files

Check how tickets are done when you change the org